[#1397] Add apphook configuration for profile page

vaszig opened the pull request:
I added the link to apphook configurations in the admin (generally) to the toolbar as it is shown below:

![2023-05-01_14-33](https://user-images.githubusercontent.com/63058071/235464234-a0435d1a-fafe-4275-8332-351f83f388e7.png)

Bartvaderkin commented on src/open_inwoner/cms/profile/cms_appconfig.py:
```diff
@@ -0,0 +1,56 @@
+
+
+class ProfileConfigForm(AppDataForm):
+    title = forms.CharField()
```
We don't use this field so let's remove it for now.

Bartvaderkin reviewed (changes requested):
As discussed in the call it would be convenient if the link to the config model's admin edit/create page could be added to the CMS toolbar if you're at the page of the apphook.

alextreme commented:
Discussed, will need rebasing due to changes on develop. Bart will double-check to determine if the configuration is in a logical place via the cms toolbar (Pagina's menu)

vaszig commented:
@Bartvaderkin I updated the branch and added a simple item to the toolbar. It's only enabled when we are on the profile page. Let me know if you agree with this approach.

![image](https://user-images.githubusercontent.com/63058071/237010471-431633fb-9c72-4168-a155-66844b61e09c.png)

alextreme commented:
Reviewed with @vaszig and approved, but double-check from @Bartvaderkin is requested due to the remarks made yesterday
